meetings/2022-03-14.md

```diff
@@ -0,0 +1,108 @@
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* elf Pavlik
+---
+
+## Agenda
+
+### Pull Requests
+https://github.com/solid/solid-oidc/pulls
+
+* 
+
+### Issues
+https://github.com/solid/solid-oidc/issues
+
+* [In some cases OIDC issuer can't be disclosed in WebID Profile #92](https://github.com/solid/solid-oidc/issues/92)
+
+
+## Minutes
+
+### DST changess
+
+* Pavlik: Can we stick with US time for next two weeks?
+* Matthieu: ok for me, even better.
+* Nicolas: ok
+
+
+### In some cases OIDC issuer can't be disclosed in WebID Profile 
+
+[Issue#92](https://github.com/solid/solid-oidc/issues/92)
+
+* MB: During AuthZ this requirement came up. We also discussed TrustedApps, which has privacy issue of having user disclosing trusted apps to the public. It stroke me that in many places disclosing trusted issuers in WebID document could be privacy issue. Depending on circumstances it could be damaging to someone forced to disclose it. For example government service.
+* eP: For example unemployment service
+* MB: We discussed someting in lines of WebID TLS where key management would be on the OP side.
+* ...: You would disclose some kind of secret allowing verifying the issuer. So pretty much just public key of the issuer.
+* NAS: So pretty much user would be providing OP rather than WebID.
+* MB: I mean that OP would keep private key for each of the user. And as a user I would disclosed the public key associated with private key used by OP for my WebID.
+* NAS: My understanding was that we want to center things around WebID. But here user would need to provide OP to the client.
+* AC: It sounds like we can use `did:key` or something described in WebID-TLS. Is it separate from Solid-OIDC? We could have different kinds of interaction which could look more like WebID-TLS. This is valuable use case but not everyone use case.
+* eP: Currently in [section 8.1](https://solid.github.io/solid-oidc/#id-token-validation), we say "unless the verifying party accepts keys via other means...". IOW, we already have this described "via other means", so could we use this to extend Solid-OIDC? Without disclosing issuer? I.e., just like Solid-OIDC but where the keys are retrieved via some other means.
+* eP: It would be useful if the keys were also in JWK, that would keep the feature very similar to the current flow
+* MB: I don't have well constructed opinion if this is too much of a departure to have it there. If it is compatible it would be nice to have it in Solid-OIDC.
```

Comment:
```suggestion
* MB: I don't have a well constructed opinion about whether this is too much of a departure to have it as part of Solid OIDC or not. If it were compatible it would be nice to have it in Solid-OIDC.
```